### ISSUE TITLE
Use of linear-gradient overrides site background color in Edge high contrast mode

### ISSUE THREAD

**Issue by jlukosky** [OfficeDev/office-ui-fabric-react#10479]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

Viewing the site in Edge (the classic one, not Chromium) while using high contrast black the page has a white background.

### Please provide a reproduction of the issue in a codepen:
1. In Windows 10, turn on high contrast using the high contrast black theme
2. Open[ Fabric site](https://developer.microsoft.com/en-us/fabric/#/controls/web/groupedlist) in Edge

 

#### Actual behavior:

Observe there is a white background. This is most noticeable in the left navigation

#### Expected behavior:

The background should match the high contrast backplate color. In this case the color should be black. 

### Documentation describing expected behavior

The following in Fabric's CSS is causing this issue:
background: linear-gradient(to right, #fff 0%, #fff 50%, #faf9f8 50%, #faf9f8 100%)

**Comment by aneeshack4** [comment]
@jlukosky Thanks for reporting this! I'm able to repro this: 
![image](https://user-images.githubusercontent.com/4161791/65083834-78e5f380-d95e-11e9-847f-e5b0b66c8dc6.png)

@Jahnp I spoke with Elizabeth & she said this was probably copied over from the HIG - maybe a HC override is appropriate here or do you know of any other fix?

**Comment by Vitalius1** [comment]
@aneeshack4 looks like this issue is not an issue anymore. This is what I see currently. @jlukosky can you please confirm that this is fixed?
![image](https://user-images.githubusercontent.com/29514833/69455658-b9c30580-0d1d-11ea-98ea-0ccae67356bb.png)
